src/neo/SmartContract/Native/LedgerContract.cs

```diff
@@ -118,30 +161,60 @@ public Block GetBlock(DataCache snapshot, UInt256 hash)
             };
         }
 
+        /// <summary>
+        /// Gets a block with the specified index.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <param name="index">The index of the block.</param>
+        /// <returns>The block with the specified index.</returns>
         public Block GetBlock(DataCache snapshot, uint index)
         {
             UInt256 hash = GetBlockHash(snapshot, index);
             if (hash is null) return null;
             return GetBlock(snapshot, hash);
         }
 
+        /// <summary>
+        /// Gets a block header with the specified hash.
+        /// </summary>
+        /// <param name="snapshot">The snapshot used to read data.</param>
+        /// <param name="hash">The hash of the block.</param>
+        /// <returns>The block header with the specified hash.</returns>
         public Header GetHeader(DataCache snapshot, UInt256 hash)
         {
             return GetTrimmedBlock(snapshot, hash)?.Header;
         }
 
+        /// <summary>
+        /// Gets a block header with the specified index.
```

Comment:
In some comments we use "Gets the xxxx with the xxxx".
Maybe we could use all like here: "Gets a xxxx with the xxx"
The "The" is quite specific for something that we are passing as parameters, it looks like the "a/an" will be more suitable.

Reply:
But I am not sure.